server/game/cards/03-WC/Velum.js

```diff
@@ -0,0 +1,34 @@
+const Card = require('../../Card.js');
+
+class Velum extends Card {
+    setupCardAbilities(ability) {
+        this.reap({
+            useActionPromptMessage: true,
```

Comment:
What's the behaviour without this flag?

Reply:
Messages without the flag (read bottom up):

```
player1 uses Velum to archive 2 cards
player1 uses Velum to archive Velum    <----- Velum is not archived. Need to remove this line
player1 uses Velum to reap with Velum
player1 plays Hyde
player1 chooses logos as their active house this turn
player1 does not forge a key.  They have 0 amber.  The current cost is 6 amber
player1: 0 amber player2: 0 amber
player2 readies all their cards
player2 chooses brobnar as their active house this turn
player2 does not forge a key.  They have 0 amber.  The current cost is 6 amber
player1: 0 amber player2: 0 amber
player1 readies all their cards
player1 chooses logos as their active house this turn
player1 does not forge a key.  They have 0 amber.  The current cost is 6 amber
player1 draws 1 cards up to their maximum hand size of 6
player2 draws 6 cards up to their maximum hand size of 6
player1 draws 6 cards up to their maximum hand size of 6
player2 is playing as the Archon: [object Object]
player1 is playing as the Archon: [object Object]
player2 is shuffling their deck
player1 is shuffling their deck
```

Messages with the flag:

```
player1 uses Velum to archive 2 cards
player1 uses Velum to reap with Velum
player1 plays Hyde
player1 chooses logos as their active house this turn
player1 does not forge a key.  They have 0 amber.  The current cost is 6 amber
player1: 0 amber player2: 0 amber
player2 readies all their cards
player2 chooses brobnar as their active house this turn
player2 does not forge a key.  They have 0 amber.  The current cost is 6 amber
player1: 0 amber player2: 0 amber
player1 readies all their cards
player1 chooses logos as their active house this turn
player1 does not forge a key.  They have 0 amber.  The current cost is 6 amber
player1 draws 1 cards up to their maximum hand size of 6
player2 draws 6 cards up to their maximum hand size of 6
player1 draws 6 cards up to their maximum hand size of 6
player2 is playing as the Archon: [object Object]
player1 is playing as the Archon: [object Object]
player2 is shuffling their deck
player1 is shuffling their deck
```